### DISCUSSION TITLE
Welcome to disposable Discussions!

### DISCUSSION THREAD

**Discussion by smeinecke** [disposable/disposable#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EeKay** [comment]
Hey what is the best way to get disposable email domains in the blacklist?

ie: meantodeal.com is a domain used by https://www.minuteinbox.com

Can you provide some guidance on the best way to fix this? 
open issue, pull request, ...

tyia

**Reply by smeinecke** [reply]
Sorry for late reply. Somehow I did not get any notification for this. Yes, PR would be best. Issue is good aswell, but I do not have the time to add new providers - even if this would be urgently necessary in the meantime. 

PRs with code to implement new hosts are very welcome :)
